app/services/suggest_random_teams_service.rb

```diff
@@ -7,6 +7,6 @@ def self.suggest
     raise NoGameInProgressError if game_in_progress.nil?
 
     players_shuffled = game_in_progress.players.map(&:slack_user_name).shuffle
-    "Randomized matchup: #{players_shuffled[0]} and #{players_shuffled[1]} vs. #{players_shuffled[2]} and #{players_shuffled[3]}"
+    "Randomized matchup: @#{players_shuffled[0]} and @#{players_shuffled[1]} vs. @#{players_shuffled[2]} and @#{players_shuffled[3]}"
```

Comment:
Because the slack API is weird this won't actually result in a @-mention, you should use this function:  https://github.com/connorproctor/foos_portal/blob/master/app/models/user.rb#L14

Reply:
ah, gotcha